alicloud-csharp/${PROJECT}.csproj

```diff
@@ -7,7 +7,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Pulumi.Alicloud" Version="1.5.0-preview" />
+    <PackageReference Include="Pulumi.Alicloud" Version="2.0.0" />
```

Comment:
I think we want to use floating versions like `2.*` to avoid changing templates frequently